addons/account/tests/test_account_account.py

```diff
@@ -53,6 +53,7 @@ def test_toggle_reconcile(self):
             ],
         })
         move.action_post()
+        self.env['account.move.line'].flush_model()
```

Comment:
Because not everything is flushed (it was with the old implementation), and a query is run with a `cr.execute` when toggling the `reconcile` flag on an account.